include/SDL3/SDL_surface.h

```diff
@@ -867,6 +879,8 @@ extern DECLSPEC int SDLCALL SDL_BlitSurfaceUncheckedScaled
 /**
  * Set the YUV conversion mode
  *
+ * \param YUV conversion mode
```

Comment:
Wrong parameter name here. Maybe change to this:
` * \param mode YUV conversion mode`

Reply:
was fixed